docs/deploy-full-rp-service-in-dev.md

```diff
@@ -49,9 +49,9 @@
     What to change in `env-int` file:
 
     * if using a public key separate from `~/.ssh/id_rsa.pub` (for ssh access to RP and Gateway vmss instances), source it with `export SSH_PUBLIC_KEY=~/.ssh/id_separate.pub`
-    * don't try to change `$USER` prefix used there
+    - Modify `AZURE_PREFIX` environment variable for a different `$AZUREPREFIX` prefix of the created Azure resources.
```

Comment:
What's the difference between those?
Having took a glance, I don't think it needs two variables.

If you are concerning about this part `export AZUREPREFIX="${AZURE_PREFIX:-$USER}"`, it can be `export AZURE_PREFIX="${AZURE_PREFIX:-$USER}"`

Reply:
I agree